homeassistant/components/airtouch5/strings.json

```
@@ -27,6 +27,11 @@
           }
         }
       }
+    },
+    "cover": {
+      "airtouch5": {
```

I'm missing a translation key in the class

```suggestion
      "damper": {
```
Also let's call it something better

---

I reused what was in the base class.

https://github.com/home-assistant/core/blob/b0b1ab317cf0dce37a85f3e1be78cf70ceda8504/homeassistant/components/airtouch5/entity.py#L12-L17

https://github.com/home-assistant/core/blob/b0b1ab317cf0dce37a85f3e1be78cf70ceda8504/homeassistant/components/airtouch5/const.py#L3

Happy to change it if you still want me too.

---

Updated translation to "domain" but left the climate translations as they were.